src/PKSim.Core/Commands/EditParameterCommand.cs

```diff
@@ -73,21 +73,21 @@ private bool isBuildingBlockParameterInSimulation(IParameter parameter)
          if (string.IsNullOrEmpty(parameter.Origin.BuilingBlockId))
             return false;
 
-         //The simulation id is not set. The parameter is defined in a building block
-         if (string.IsNullOrEmpty(parameter.Origin.SimulationId))
-            return false;
-
-         //a parameter defined in a simulation? Not a building block parameter in a  simulation
-         if (parameter.BuildingBlockType == PKSimBuildingBlockType.Simulation)
+         //THIS SHOULD NOT HAPPEN and should be deleted
+         if (string.IsNullOrEmpty(parameter.Origin.SimulationId) || parameter.BuildingBlockType == PKSimBuildingBlockType.Simulation)
+         {
+            Debug.Print($"This should not happen for parameter {parameter.Name}");
```

Comment:
If we don't see it happen, we should simply remove this check in the future. Keep for now